index.html

```diff
@@ -0,0 +1,131 @@
+  <link rel="stylesheet" href="./styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PoPraX</title>
+</head>
+<body>
+  <main>
+    <div class="top-top-nav desktop">
+      <ul>
+        <li><i class="social-media-icons fa-brands fa-twitter"></i></li>
+        <li><i class="social-media-icons fa-brands fa-facebook-f"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="top-nav desktop">
+      <a href="#home" class="logo">PoPraX</a>
+      <ul>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#top-songs">Top songs</a></li>
+        <li><a href="#featured-performers">Performers</a></li>
+        <li><a href="#home">PoPraX</a></li>
+      </ul>
+    </nav>
+    <header id="home">
+      <i class="fa-solid fa-bars menu-icon mobile"></i>
+      <div class="heading-content">
+        <h1>The cheapest tickets on the internet, period.</h1>
+      <p>Zero hidden service fees, price match guarantee, group discounts, and more. Beacuse it <b>Should</b> be this easy</p>
+      <a class="events-btn" href="#">Checkout upcoming events </a>
+      <div id="side-menu" class="menu-items">
+        <i class="fa-solid fa-xmark closebtn close-menu mobile"></i>
+        <div class="overlay-content">
+          <a href="about.html" class="close-menu">About</a>
+          <a href="#top-songs" class="close-menu">Top Songs</a>
+          <a href="#featured-performers" class="close-menu">Performers</a>
+          <a href="#partners" class="close-menu">Partners</a>
+        </div>
+      </div>
+      </div>
+    </header>
+
+    <section id="top-songs" class="top-songs">
+      <h2>Top songs worldwide</h2>
+      <div class="musics-container">
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/harry-styles-bma-as-it-was-po3-344x344.jpg" alt="">
+            <span class="music-author">As It Was</span>
+            <p class="music-description">Harry Styles</p>
+          </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/lizzo-o9x-about-damn-time-f6q-344x344.jpg" alt="">
+            <span class="music-author">Lizzo </span>
+            <p class="music-description">About Damn Time </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/jack-harlow-i43-344x344.jpg" alt="">
+            <span class="music-author">Jack Harlow </span>
+            <p class="music-description">First Class </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/kate-bush-qv2-running-up-that-hill-1cx-344x344.jpg" alt="">
+            <span class="music-author">Kate Bush </span>
```

Comment:
- [ ] `required`
- [ ] just as mentioned on top I suggest you follow the original template here as well. please see screenshots below

`figma template`
![main program](https://user-images.githubusercontent.com/66230306/181393566-9ee0abc1-e275-4e9f-89da-7c0739213ce7.png)

`your template`
![main polo](https://user-images.githubusercontent.com/66230306/181393623-1c36324b-a88b-4665-815f-389f94fc5011.png)